packages/flutter_sprites/lib/src/virtual_joystick.dart

```diff
@@ -31,18 +31,18 @@ class VirtualJoystick extends NodeWithSize {
   Paint _paintControl;
 
   bool handleEvent(SpriteBoxEvent event) {
-    if (event.type == "pointerdown") {
+    if (event.type == PointerDownEvent) {
```

Comment:
"event is PointerDownEvent"?  I'm not entirely sure why this isn't giving an analyzer warning.